pkg/csi/cinder/driver.go

```diff
@@ -32,6 +32,12 @@ import (
 const (
 	driverName  = "cinder.csi.openstack.org"
 	topologyKey = "topology." + driverName + "/zone"
+
+	// MaxVolumesPerNode is the maximum number of volumes that can be attached to a node
+	MaxVolumesPerNode = 256
```

Comment:
Where does this const come from?

Reply:
It was hardcoded here https://github.com/kubernetes/cloud-provider-openstack/blob/d069b1ef5d34e1400c798424f5d56b11098fb9a7/pkg/csi/cinder/openstack/openstack.go#L144

Reply:
just in case, if the const/var doesn't need to be accessed from a different package, it's better to define its name starting with a lower case (non-exported). https://go.dev/tour/basics/3

Reply:
Yep, `--node-volume-attach-limit` flag customize it

Reply:
MaxVolumesPerNode -> maxVolumesPerNode 👍